pipeline/inputs/standard-input.md

```diff
@@ -1,18 +1,37 @@
 # Standard Input
 
-The **stdin** plugin allows to retrieve valid JSON text messages over the standard input interface \(stdin\). In order to use it, specify the plugin name as the input, e.g:
+The **stdin** plugin allows to retrieve a message stream from the standard input interface \(stdin\) of the fluent-bit process. In order to use it, specify the plugin name as the input, e.g:
```

Comment:
Let's ensure we use `Fluent Bit` everywhere and split sentences to simplify reviews.

```suggestion
The **stdin** plugin supports retrieving a message stream from the standard input interface \(stdin\) of the Fluent Bit process.
In order to use it, specify the plugin name as the input, e.g:
```